### PR TITLE
.github/ISSUE_TEMPLATE: Suggest '/kind bug' and '/kind feature'

### PR DESCRIPTION
**Pull request by wking**
So [Prow's label plugin][1] can apply the appropriate label for us.

/kind feature

For an example, of `/kind bug` working, see [here][2].

[1]: https://github.com/kubernetes/test-infra/tree/master/prow/plugins/label
[2]: https://github.com/containers/libpod/issues/2086#event-2055746479

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -18,13 +18,11 @@ executes Buildah to perform container builds, and as such the Buildah
 maintainers are best equipped to handle these bugs.
 -->
 
-**Is this a BUG REPORT or FEATURE REQUEST?**:
+**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**
 
-[//]: # Uncomment only one, leave it on its own line:
+/kind bug
```
[the remark this answers is not in the document]
Comment (wking) [reply .github/ISSUE_TEMPLATE.md]: > What will the bot do if both are present? Add both labels?

Yeah, probably.  Do you expect cleaning up after that to be more or less work than just adding the labels yourself? ;)

**Comment by mheon** [comment]
@wking Mind rebasing to pick up CI fixes?

**Comment by wking** [comment]
> Mind rebasing to pick up CI fixes?

Rebased onto master (no conflicts) with 029afca -> 99e642d.

**Pull request merged**